tests/unit/core/agents/agent_components/test_streaming_debug.py

```
@@ -0,0 +1,476 @@
+        assert content_preview is None
+
+    def test_delta_without_content_delta(self):
+        event = SimpleNamespace(delta=SimpleNamespace())
+        delta_type, content_len, content_preview = _get_delta_debug_data(event)
+        assert delta_type == "SimpleNamespace"
+        assert content_len is None
+        assert content_preview is None
+
+    def test_delta_with_non_string_content(self):
+        event = SimpleNamespace(delta=SimpleNamespace(content_delta=42))
+        delta_type, content_len, content_preview = _get_delta_debug_data(event)
```

Ruff (0.14.14)

[warning] Unpacked variable `content_preview` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)